### ISSUE TITLE
Logo only buttons

### ISSUE THREAD

**Issue by zlace** [beerstorm-net/the_apple_sign_in#5]
Would love to have to option for a Logo only button style. Would be happy to contribute a PR if needed.

https://developer.apple.com/design/human-interface-guidelines/sign-in-with-apple/overview/buttons/

**Comment by zeusbaba** [comment]
that's a good suggestion.  
you are welcome to contribute with a PR.

**Comment by ZaaWi** [comment]
Yes I agree, as this would greatly assist us developers with future projects. Would love to contribute if needed.
